docs/development/core/public/kibana-plugin-public.i18nsetup.context.md

```diff
@@ -0,0 +1,13 @@
+[Home](./index) &gt; [kibana-plugin-public](./kibana-plugin-public.md) &gt; [I18nSetup](./kibana-plugin-public.i18nsetup.md) &gt; [Context](./kibana-plugin-public.i18nsetup.context.md)
+
+## I18nSetup.Context property
+
+React Context provider required as the topmost component for any i18n-compatible React tree.
+
+<b>Signature:</b>
+
+```typescript
+Context: ({ children }: {
+        children: React.ReactNode;

```

Comment:
nit: indentation consistency.

```suggestion
  children: React.ReactNode;
```

Reply:
this file is autogenerated. the problem should be fixed in https://github.com/Microsoft/web-build-tools/tree/master/apps/api-documenter 
@skaapgif did you create an issue for this problem or I should?